dashboard/app/views/home/index.html.haml

```diff
@@ -4,22 +4,18 @@
   = tag 'meta', property: 'og:description', content: t('share.description')
 
 
-- show_ui_tips = false
-
 -# Student modal
 - if (current_user && current_user.show_race_interstitial?) || @force_race_interstitial
   = render partial: 'layouts/race_interstitial'
 
 -# Teacher modals
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   = render partial: 'layouts/terms_interstitial'
-- elsif (current_user && current_user.show_school_info_interstitial?) || @force_school_info_interstitial
+- if (current_user && current_user.show_school_info_interstitial?) || @force_school_info_interstitial
```

Comment:
I think this should remain `elsif` so that we don't show more than one interstitial.